metrique-metricsrs/src/accumulator.rs

```diff
@@ -208,8 +207,8 @@ impl<V: MetricsRsVersion + ?Sized> MetricAccumulatorEntry<V> {
     }
 
     /// Get the current timestamp from this metrics accumulator entry
-    pub fn timestamp(&self) -> Option<SystemTime> {
-        self.timestamp
+    pub fn timestamp(&self) -> Option<&metrique_timesource::SystemTime> {
```

Comment:
this should probably return a normal system time if this is actually a pub fn

Reply:
although if we call it internally, returning the metrique one allows calling `elapsed`

Reply:
we don't call `elapsed`, but I think it's better not to destroy information. Nobody really calls this function externally.